app/plugin_test.go

```diff
@@ -50,17 +50,15 @@ func TestPluginKeyValueStore(t *testing.T) {
 
 	// Test ListKeys
 	assert.Nil(t, th.App.SetPluginKey(pluginId, "key2", []byte("test")))
-	hashedKey := getHashedKey("key")
-	hashedKey2 := getHashedKey("key2")
 	list, err := th.App.ListPluginKeys(pluginId, 0, 1)
 	assert.Nil(t, err)
 	assert.Equal(t, 1, len(list))
-	assert.Equal(t, hashedKey, list[0])
+	assert.Equal(t, "key", list[0])
```

Comment:
Can we add some tests for the fallback behavior, in case we have a hashed key already in the database and query it using the normal key, and the data is returned correct? We can enforce the old value creation using `th.App.Srv.Store.Plugin().SaveOrUpdate(kv)` in a test, with a KV struct where the key is already hashed and then query it afterwards. Similar, we should test the `Set` method replacing the old version.